README.md

```diff
@@ -1,166 +1,171 @@
+  name: 'my-component',  
+  methods: {  
     login () {
-      this.$getGapiClient()
-        .then(gapi => {
-          // gapi.sheets.spreadsheet.get(...)
-          // ...
-        })
+      this.$gapi.getGapiClient()  
+        .then(gapi => {  
+          // gapi.sheets.spreadsheet.get(...
+        }
+        .catch(err => {
+	      // Handle errors
+        }
     }
-  },
-
+  }
 }
-</script>
-```
-
-## Login
-
-This will shortcut the login process
-
-```html
-<script>
-export default {
-  name: 'login-shortcut',
-
-  methods: {
-    login () {
-      this.$login()
+</script>  
+```  
+  
+## Login  
+Initiate the login process.
+  
+```html  
+<script>  
+export default {  
+  name: 'login',
+  methods: {  
+    login () { 
+       this.$gapi.login()
+         .then(() => {
+           // Any actions needed after login
+         }
+         .catch(err => {
+	       // Handle errors
+         }
     }
   }
-
 }
-
 </script>
-```
-
-## Logout
-
-This will shortcut the logout process
-
-```html
-<script>
-export default {
-  name: 'logout-shortcut',
-
+```  
+  
+## Logout  
+Initiate the logout process.
+  
+```html  
+<script>  
+export default {  
+  name: 'logout',
   methods: {
-    logout () {
+    logout () { 
       this.$logout()
+        .then(() => {
+          // Any actions needed after logout
+        }
+        .catch(err => {
+          // Handle errors
+        }
     }
   }
-
 }
-
-</script>
-```
-
-## isAuthenticated
-
-This will shortcut and check if your user is authenticated will return a boolen
-
-```html
-<script>
+</script>  
+```  
+  
+## isAuthenticated  
+Use isAuthenticated to check if a user is authenticated. It returns true if the user is authenticated, otherwise false.
+
+```html  
+<script>  
 export default {
-  name: 'login-shortcut-check',
-
+  name: 'auhtenticated-check',
   methods: {
     login () {
-      if (this.$isAuthenticated() !== true) {
-        this.$login()
+      if (this.$gapi.isAuthenticated()) {  
+        this.$login() 
+          .then(() => {
+            // Any actions needed after login
+          }
+          .catch(err => {
+ 	        // Handle errors
+          }
       }
     }
   }
-
 }
-
-</script>
+</script>  
 ```
-
-## refreshToken
-
-This will shortcut getting a refresh token from Google, this should be placed in your App.vue on the created page and run on a timer of 45min
-
-```html
-<script>
-  name: 'App'
-
+## refreshToken  
+This will shortcut getting a refresh token from Google, this should be placed in your App.vue on the created page and run on a timer of 45min  
+  
+```html  
+<script>  
+export default {
+  name: 'App',
   created () {
-  try {
-    // NOTE: 45min refresh policy is what google recommends
-    window.setInterval(this.$refreshToken(), 1000 * 60 * 45)
-  } catch (e) {
-    console.error(e)
+    try {
+      // NOTE: 45 min refresh policy is what Google recommends
+      window.setInterval(this.$refreshToken(), 1000 * 60 * 45)  
```

Comment:
this.$gapi.refreshToken

Reply:
Yes you are right. Missed this one. Thanks for catching it.